src/components/StaffList/StaffList.test.jsx

```diff
@@ -27,18 +30,89 @@ const input = {
   ],
   onChange: jest.fn(),
 };
+const autoCompletePersonResponses = {
+  long: [
+    {
+      id: '5',
+      text: {
+        uuid: 'a7d0e2c0-9a02-421b-93bf-d081339090cc',
+        profile_image_url: '/assets/new-80.png',
+        given_name: 'Pippi',
+        family_name: 'Longstocking',
+      },
+    },
+    {
+      id: '6',
+      text: {
+        uuid: 'b7d0e2c0-9a02-421b-93bf-d081339090cc',
+        profile_image_url: '/assets/new-80.png',
+        given_name: 'Hank',
+        family_name: 'Longfellow',
+      },
+    }],
+};
 
 describe('StaffList', () => {
   afterEach(() => {
     // Clear onChange's call count after each test
     input.onChange.mockClear();
+    // reset api client response
+    mockClient.reset();
   });
 
-  it('renders a grid of staff members', () => {
+  it('renders a grid of staff members and an autocomplete search input', () => {
     const component = shallow(<StaffList input={input} />);
     expect(component).toMatchSnapshot();
   });
 
+  it('gets/clears suggestions for autocomplete', (done) => {
+    mockClient.onGet('http://localhost:18381/admin/course_metadata/person-autocomplete/?q=long&serialize=1')
+      .replyOnce(200, JSON.stringify({
+        results: autoCompletePersonResponses.long,
+      }));
+    const component = mount(<StaffList input={input} />);
+    component.instance().onSuggestionsFetchRequested({ value: 'long' }).then(() => {
+      let state = component.state().suggestions;
```

Comment:
I would do `const { suggestions } = component.state()` here, if you're trying to check what the suggestions are

Reply:
took your suggestion (mostly) but had to make it a 'let' as I need to refresh the value later.

Reply:
Actually, could not get any version of this passed the linter.

Reply:
Huh that's weird. I can (probably) help with the linting, it's just usually nicer to use destructuring, and `state` doesn't feel like a great variable

Reply:
👍